in_toto/models/metadata.py

```diff
@@ -282,3 +291,28 @@ def _validate_signatures(self):
 
     for signature in self.signatures:
       securesystemslib.formats.ANY_SIGNATURE_SCHEMA.check_match(signature)
+
+
+def dump_metadata(path, metadata, compact_json=False):
+  """A temporary function that dumps metadata json into a file."""
+
+  indent = 1
+  separators = (',', ': ')
+  if compact_json:
+    indent = None
+    separators = (',', ':')
```

Comment:
Note that `Metablock` also has a `compact_json` attribute, which is ignored here. This should really only be a temporary solution. Adding a TODO comment or creating a GitHub issue might help to not forget.